charts/skypilot/templates/api-deployment.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: skypilot-api-server
```

Comment:
Minor: shall we prefix all resource names by `{{ .Release.name }}` to avoid collision?

Reply:
good idea, thanks!